package.json

```
@@ -88,7 +88,7 @@
     "@expensify/react-native-background-task": "file:./modules/background-task",
     "@expensify/react-native-hybrid-app": "file:./modules/hybrid-app",
     "@expensify/react-native-live-markdown": "0.1.302",
-    "@expensify/react-native-wallet": "^0.1.5",
+    "@expensify/react-native-wallet": "^0.1.10",
```

Can we get rid of the `^` symbol? Personally I believe that it shouldn't be used on such big codebase in general
```suggestion
    "@expensify/react-native-wallet": "0.1.10",
```